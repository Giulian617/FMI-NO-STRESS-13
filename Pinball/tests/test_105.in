100 10265
32 82 2
27 82 6
16 82 2
79 82 7
82 94 1
29 82 8
82 18 8
48 82 9
82 68 6
22 82 10
80 82 2
82 67 9
58 82 5
14 82 7
82 66 5
82 83 9
76 82 5
82 20 2
15 82 5
82 71 4
86 82 5
74 82 9
91 82 9
69 82 7
82 78 1
6 82 3
70 82 8
82 99 9
82 3 10
59 82 2
82 93 9
61 82 4
82 4 10
82 12 8
82 65 6
2 82 2
8 82 3
82 33 4
82 98 8
54 82 7
82 34 4
82 62 5
82 30 4
82 63 3
25 82 7
82 37 1
82 45 7
82 40 7
88 82 9
82 64 1
82 46 3
87 82 3
47 82 2
73 82 1
82 10 7
82 96 5
7 82 9
82 53 8
82 42 2
82 89 2
92 82 1
72 82 10
82 90 7
82 9 1
13 82 10
82 57 6
60 82 3
1 82 1
82 28 7
82 41 4
84 82 9
44 82 5
77 82 3
19 82 3
82 21 9
23 82 1
82 5 10
39 82 6
82 35 4
82 56 9
82 51 5
100 82 10
82 97 4
81 82 9
85 82 3
82 24 2
75 82 7
50 82 7
49 82 4
82 17 7
95 82 7
36 82 5
38 82 5
31 82 9
82 11 6
82 26 10
82 43 3
82 55 4
52 82 2
