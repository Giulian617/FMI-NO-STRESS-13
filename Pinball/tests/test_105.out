575723126
158693272
163283177
163283177
163283177
636555840
455965842
636555840
959584557
963758616
124340488
709913649
163283177
963758616
946605057
158693272
963758616
709913649
636555840
158693272
455965842
163283177
959584557
158693272
963758616
163283177
124340488
963758616
709913649
703582817
455965842
158693272
703582817
703582817
703582817
946605057
959584557
946605057
124340488
963758616
703582817
158693272
636555840
946605057
963758616
636555840
158693272
455965842
703582817
963758616
946605057
158693272
709913649
963758616
703582817
455965842
124340488
946605057
158693272
636555840
703582817
946605057
636555840
959584557
124340488
946605057
455965842
124340488
963758616
709913649
703582817
163283177
959584557
455965842
963758616
946605057
636555840
959584557
963758616
158693272
455965842
455965842
455965842
636555840
946605057
636555840
455965842
158693272
963758616
455965842
959584557
455965842
959584557
963758616
946605057
703582817
709913649
455965842
163283177
